test/custom-parser.test.js

```diff
@@ -8,6 +8,29 @@ const sget = require('simple-get').concat
 const Fastify = require('..')
 
 const jsonParser = require('fast-json-body')
+// Adapted from https://github.com/delvedor/fast-json-body/blob/master/json-body.js
```

Comment:
can you add a space before the comment?

Reply:
I dont think comment is necessary (imo)

Reply:
@mcollina @cemremengu should I remove the source credit comment?

Reply:
Since @delvedor is a core mainter of fastify I dont think it will be a problem for him :)

Reply:
;-)

Reply:
done and comment removed.